api/src/plugins/csrf.ts

```diff
@@ -23,7 +23,7 @@ const csrf: FastifyPluginCallback = (fastify, _options, done) => {
     // tokens since we know we can provide this one
     getToken: req => req.headers[CSRF_HEADER] as string,
     cookieOpts: { signed: false, sameSite: 'strict' },
-    logLevel: 'debug' // TODO: change to trace
+    logLevel: 'trace' // TODO: change to silent once that's supported.
```

Comment:
If this was essentially hard-coded before then how did `FCC_API_LOG_LEVEL` affect this before this PR?

Reply:
`FCC_API_LOG_LEVEL` sets the minimum level that's logged.

e.g. if it's `info` then `debug` and `trace` don't get logged.

Reply:
Sorry -- I am not getting how the variable is connected to this line of code?

Reply:
Fastify plugins, like this one, allow you to configure their logging via the `logLevel` option:

https://github.com/fastify/csrf-protection/blob/032345a50078f35739b85e98429fcd425ce62a39/index.js#L119

The plugin calls `fastify.log.debug('Missing csrf secret')` if we set the `logLevel` to `'debug'`.

So, what's the connection to `FCC_API_LOG_LEVEL`? It's the same as if we'd called `fastify.log.debug('Missing csrf secret')` in our code. i.e. If the `FCC_API_LOG_LEVEL` is `debug` or `trace` it appears in the logs, otherwise not.